<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>autoCCode</class>
 <widget class="QWidget" name="autoCCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>autoCCode</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="get_listlabel">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>GetList</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="db_comboBox"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="codeshow_textEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="tocode_btn">
             <property name="text">
              <string>To代码库</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="save_btn">
             <property name="text">
              <string>save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="close_btn">
             <property name="text">
              <string>close</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="genshow_textEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
